src/control/lib/hostlist/numericlist_test.go

```diff
@@ -0,0 +1,106 @@
+//
+// (C) Copyright 2019 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+package hostlist_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/daos-stack/daos/src/control/lib/hostlist"
+)
+
+func TestHostList_CreateNumericList(t *testing.T) {
+	for name, tc := range map[string]struct {
+		startList    string
+		expRawOut    string
+		expUniqOut   string
+		expUniqCount int
+		expErr       error
+	}{
+		"simple": {
```

Comment:
name nit - maybe "letters in numeric list" or something else indicating the expected error?